### PR TITLE
Correct mismatch between allowed and adjusted tuning

### PR DESCRIPTION
**Pull request by kergoth**

### CODE REVIEW

File: meta-sokol-flex-distro/conf/distro/sokol-flex.conf [hunk 1]
```diff
@@ -203,7 +203,7 @@ INHERIT += "codename_is_corename"
 
 # Ensure we're building the toolchain in a supported configuration
 ERROR_QA:append = " disallowed-tuning"
```
[the remark this answers is not in the document]
Comment (kergoth) [reply meta-sokol-flex-distro/conf/distro/sokol-flex.conf:205]: If `ALLOWED_TUNING` is empty, the check will always succeed, but not adding it in the first place works too.

File: meta-sokol-flex-distro/conf/distro/sokol-flex.conf [hunk 2]
```diff
@@ -203,7 +203,7 @@ INHERIT += "codename_is_corename"
...
-ALLOWED_TUNING:feature-flex-bsp = "armv5te armv7ahf-neon armv7a-neon armv7at armv8a i686 riscv64 x86-64"
+ALLOWED_TUNING:feature-flex-bsp = "armv5te armv7ahf-neon armv7a-neon armv7at armv8a i686 riscv64 x86_64"
```
[the remark this answers is not in the document]
Comment (kergoth) [reply meta-sokol-flex-distro/conf/distro/sokol-flex.conf:206]: It's the difference between DEFAULTTUNE and TUNE_PKGARCH. Currently it checks against the latter, which uses the underscore, not the former, which uses the dash.

[the remark this answers is not in the document]
Comment (muhamzai) [reply meta-sokol-flex-distro/conf/distro/sokol-flex.conf:206]: got it, thanks

File: meta-sokol-flex-distro/conf/distro/sokol-flex.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (kergoth) [reply meta-sokol-flex-distro/conf/distro/sokol-flex.conf:205]: Nevermind, I see that the default value isn't empty, so it'll actually fail instead!

[the remark this answers is not in the document]
Comment (kergoth) [reply meta-sokol-flex-distro/conf/distro/sokol-flex.conf:205]: Fixed now

**Review by muhamzai — APPROVED** [review]

**Review by abelal — APPROVED** [review]

**Pull request merged**